physics/module_sf_mynn.F90

```diff
@@ -1750,15 +1775,16 @@ SUBROUTINE SFCLAY1D_mynn(                                      &
          ! CALCULATE THE EXCHANGE COEFFICIENTS FOR HEAT (FLHC)
          ! AND MOISTURE (FLQC)
          !------------------------------------------
+         !tgs - should FLQC, FLHC be separate for dry, icy and wet?
```

Comment:
All of the rest of the questions are up to us. If we go back to using the fluxes, which I think we want to do, then yes, we should break everything up into different types.

Reply:
I think we should go back to MYNN surface fluxes over water.